WooCommerce/Classes/POS/Presentation/Reusable Views/POSModalViewModifier.swift

```diff
@@ -90,8 +33,6 @@ extension View {
                                       @ViewBuilder content: @escaping () -> ModalContent) -> some View {
         self.modifier(
             POSModalViewModifier(isPresented: isPresented,
-                                 modalContent: content,
-                                 fixedWidth: fixedWidth,
-                                 fixedHeight: fixedHeight))
```

Comment:
Should we remove fixedWidth and fixedWidth as parameters?

Reply:
Yes, I just didn't want to until after the merge, so thanks for the reminder.

Done in fb546e1